discovery/spec.md

```diff
@@ -783,54 +783,47 @@ upon return of the `202 Accepted` from the original request.
 #### `POST /services`
 
 This MUST update the collection of Services in the Discovery Endpoint to
-contain all Services in the request or fail. Any existing Service identified
-in the request will be completely replaced by the Service definition in the
-request. The body of the request message MUST contain an array of zero or more
-Service entries.
-
-The `id` and `epoch` attributes of the Services MAY be omitted. In such cases
-the discovery endpoint MUST assign an appropriate value to the omitted
-field(s). An appropriate `id` MUST be globally unique. An appropriate `epoch`
-MUST be greater than the existing `epoch` for a service with the same `id`.
-When an `id` is provided any existing Service definition with that
-`id` MUST be completely replaced by the Service definition in the request.
-If an `epoch` is given in the request it MUST be greater than the
-`epoch` of any existing Service of the same `id` or the request MUST fail.
-
-Aside from `id` and `epoch`, unless there is an out of band agreement, all
-mandatory attributes MUST be present in the request, and a Discovery Endpoint
-MUST reject requests that are missing such attributes.
-
-If the Discovery Endpoint is unable to successully add every Service in the
-incoming request then an error MUST be generated and none of the specified
-Services are to be added or updated in the Discovery Endpoint.
-
-The Services in the request MUST be processed in the order in which they are
-given. While any error during the update MUST cause the rollback of any effect
-of the request, the processing of each Service in the request MUST be
-applied as if all previous Services in the request were already completed.
-This means that attribute uniqueness checking, such as on the `name`
-attribute, MUST be done on the new state of all Services as if all previously
-specified Services in the incoming request were already successfully processed.
-
-For example, if a Discovery Endpoint has a Service called `MyService`, it is
-possible to rename that Service to `YourService` and then to add, or update,
-another Service to use `MyService` as its `name`.
-
-Likewise, while it might be less than optimal, it is possible for
-a request to update a Service multiple times if the same `id` is used more
-than once within a request.
+contain all Services in the request or fail. The body of the request message
```

Comment:
This seems like a pretty big shift in the behavior of this endpoint.  More specifically, it seems, by 798 and 803 that import is blocked and I don't think we discussed or found agreement on eliminating that use case.

Reply:
I think I mentioned on the call that I removed import logic because I wanted to do it in a follow-on PR. I wanted to focus on the simple cases first.  Is there another aspect that's part of the "big shift" you're thinking of?